core/invoker/src/main/scala/whisk/core/invoker/ActionLogDriver.scala

```diff
@@ -32,19 +32,19 @@ import whisk.http.Messages
 /**
  * Represents a single log line as read from a docker log
  */
-protected[invoker] case class LogLine(time: String, stream: String, log: String) {
+protected[core] case class LogLine(time: String, stream: String, log: String) {
```

Comment:
I was able to revert the scope back to `invoker`. The wider scope must have been an intermediate work result. Good finding.